packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -361,9 +363,15 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
 
     openFile(uri: URI): void {
         const options = this.buildOpenerOptions();
-        const resolvedOpener = this.openerService.getOpener(uri, options);
-        resolvedOpener
+        const closedEditor = this.navigationLocationService.closedEditorsStack.find(editor => editor.uri === uri);
```

Comment:
I'd move this query into the `then(widget =>` callback.

Reply:
So we can't move the callback there, because opening the given `uri` and opening the editor will remove it from the `closedEditorsStack`.